src/main/java/org/isf/lab/gui/LabEditExtended.java

```diff
@@ -627,7 +627,6 @@ private JButton getOkButton() {
 				}
 				List<String> labRow = new ArrayList<>();
 				lab.setDate(TimeTools.getNow());
```

Comment:
this should be `lab.setDate(examDate);` as previously set from the `examDateFieldCal`

Reply:
Yes this is an issue that I'm pushing now